_posts/2021-12-21-wt-open-research-fund.md

```diff
@@ -0,0 +1,47 @@
+
+The full proposal (personal information redacted) can be found on Zenodo: [DOI: 10.5281/zenodo.5267934](https://zenodo.org/record/5267934). Below we share some highlights from the proposal and OLS plans for the next two years.
+
+## Goals and Outcomes
+
+Our main goal with this proposal is to conduct a research-based assessment, decolonisation and dissemination of our resources for their adoption and reuse across different open science communities. A successfully completed Open Research Fund activity in OLS will enable:
+
+1. research-based evaluation and enhancement of the OLS curriculum, training materials, mentoring formats and value-based approach to local community development,
+1. incentivised integration of contextual (sociotechnical and cultural) knowledge of mentors and project leads in the open science project they develop in the program,
+1. enhanced representation and equitable participation of health researchers and community builders from the Global South in a shared vision for the global open science through collaboration,
+1. assessment of the long-term impact of OLS on the careers of the OLS participants and adoption of contextualised open science practices in research work in their regional contexts across low- and middle-income, as well as high-income countries, and
+1. publication of a series of reports capturing different outputs, building an open source evidence base and inviting support for the sustainability of OLS over the next 5 years.
+
+## Funded Position: Programme Coordinator and Researcher
+
+To maximise the impact of our project, we will recruit a staff member to join the OLS team as a Programme Coordinator and Researcher. They will also provide administrative support such as by helping ensure transparent communication with all the stakeholders, participating in outreach efforts, processing feedback from the cohorts and developing reports to share openly. Working with the OLS leadership team, they will also co-lead the impact studies and evaluation of OLS training and resources. A job announcement will be made in February 2022. 
+
+## Steering Committee
+
+Drawing from previous graduates, existing collaborators and volunteers in the OLS community, we will establish a steering committee. The committee will be engaged in co-creating a sustainability model for OLS beyond the lifetime of this funding as well as beyond the mentoring period of OLS projects (potentially in their local communities). The OLS team rely on continuous feedback from the OLS community and support to construct these pathways such that they are and will remain meaningful for them.
```

Comment:
I've tried to reword 

```
 and support to construct these pathways such that they are and will remain meaningful for them.
```

But I don't really understand what it's trying to say, can we rephrase?

Reply:
it's taken as is from the application - I think Emmy wrote this bit.

Reply:
```suggestion
Drawing from previous graduates, existing collaborators and volunteers in the OLS community, we will establish a steering committee. The committee will be engaged in co-creating a sustainability model for OLS beyond the lifetime of this funding as well as beyond the mentoring period of OLS projects (potentially in their local communities). The OLS team will continue to seek feedback from the OLS community to improve pathways for engagement that are meaningful for all OLS participants.
```

Reply:
I thought you might say that 😂